client/src/pages/tasks/Show.js

```diff
@@ -278,7 +284,7 @@ const TaskShow = ({ pageDispatchers }) => {
                       )
                     }
                   />
-                  {Settings.fields.task.parentTask && (
+                  {Settings.fields.task.parentTask && task?.parentTask?.uuid && (
```

Comment:
Actually, task is never null/undefined here, so `task.parentTask?.uuid` is also fine.